### ISSUE TITLE
Support building for web with emscriptem

### ISSUE THREAD

**Issue by Gouellie** [raylib-extras/game-premake#25]
Some indications on how to setup the project to compile for web with emscriptem would be appreciated.

**Comment by Peter0x44** [comment]
I don't believe this is currently supported, hence there are no instructions for it.

**Comment by JeffM2501** [comment]
Premake does not support a web build out of the box, so game-premake doesn't do it. You would use the normal makefile process for that.
